src/ui/Interface.js

```diff
@@ -19,8 +19,22 @@ module.exports = function Interface (options) {
 
   panMapWhenInputsChange();
 
+
+  onDrag = options.onDrag || function onDrag() {
```

Comment:
maybe this should be `options.onDrag = options.onDrag || function onDrag() {`

that way you can still override `options.onDrag` externally... not sure if you could do that for simply `onDrag` and also that'd be a global i think...